ci-operator/jobs/infra-periodics.yaml

```diff
@@ -31,7 +31,7 @@ periodics:
       - --confirm
       command:
       - /bin/commenter
-      image: docker-registry.default.svc:5000/ci/commenter:latest
+      image: gcr.io/k8s-testimages/commenter:v20181130-d7867ec
```

Comment:
Hm, will we need to change the version tags everywhere when we want to bump? How often do we bump?